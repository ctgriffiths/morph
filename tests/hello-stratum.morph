{
    "name": "hello",
    "kind": "stratum",
    "chunks": {
        "hello": {
            "repo": "hello",
            "ref": "master"
        }
    }
}
