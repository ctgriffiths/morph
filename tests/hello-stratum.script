#!/bin/sh
#
# Test building a stratum for GNU Hello.
#
# Copyright (C) 2011  Codethink Limited
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; version 2 of the License.
# 
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License along
# with this program; if not, write to the Free Software Foundation, Inc.,
# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

set -e

tar -C "$DATADIR" -xf tests/hello-chunk.tar.gz

./morph --no-default-configs build "tests/hello-stratum.morph" \
    --git-base-url "file://$DATADIR/" \
    --cachedir="$DATADIR"
tar -tf "$DATADIR/"*.stratum | LC_ALL=C sort
