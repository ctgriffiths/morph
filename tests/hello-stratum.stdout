./
baserock/
baserock/hello.meta
usr/
usr/local/
usr/local/bin/
usr/local/bin/greeter
